clients/apple/lockbook.xcodeproj/project.pbxproj

```diff
@@ -810,7 +820,7 @@
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 2;
 				DEAD_CODE_STRIPPING = YES;
-				DEVELOPMENT_TEAM = 39ZS78S25U;
+				DEVELOPMENT_TEAM = 4PN4ZH29UT;
```

Comment:
We probably want to reset this back to the original value (we need to give you our team account access).